recipes/spy/0.0.3/conanfile.py

```diff
@@ -0,0 +1,33 @@
+import os
+from conans import ConanFile, tools
+
+
+class SpyConan(ConanFile):
+    name = "spy"
+    settings = "os", "compiler", "build_type", "arch"
+    version = "0.0.3"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://jfalcou.github.io/spy/"
+    description = "C++ 17 for constexpr-proof detection and classification of informations about OS, compiler, etc..."
+    topics = ("c++17", "config", "metaprogramming")
+    no_copy_source = True
+    scm = {
+        "type": "git",
+        "url": "auto",
+        "revision": "auto"
+    }
```

Comment:
```suggestion
```
SCM attribute does not make sense for recipes hosted on CCI, the sources are not on the same repo

Reply:
dully noted